docfx/docs/guides/getting-started.md

```diff
@@ -5,31 +5,52 @@ description: How to get started installing & using CounterStrikeSharp.
 
 # Getting Started
 
-How to get started installing & using CounterStrikeSharp.
+In this guide you will learn how to install CounterStrikeSharp onto your vanilla Counter-Strike 2 server. `CounterStrikeSharp` uses `Metamod:Source` as its main way of communicating with the game server, so both frameworks will need to be installed.
+
+If you're more of a visual person, here is a <a href="https://www.youtube.com/watch?v=FlsKzStHJuY" target="_blank">Youtube video</a> that covers everything.
+
+## Prerequisites
+- <a href="https://www.sourcemm.net/downloads.php/?branch=master" target="_blank">Metamod: Source 2.X Dev Build</a>
+- <a href="https://github.com/roflmuffin/CounterStrikeSharp/releases" target="_blank">CounterStrikeSharp With Runtime</a>
 
 ## Installing Metamod
 
-`CounterStrikeSharp` uses `Metamod:Source` as its main way of communicating with the game server. To install it, you can follow the detailed instructions found <a href="https://cs2.poggu.me/metamod/installation/" target="_blank">here</a>.
+1. Extract Metamod and copy the `/addons/` directory to `/game/csgo/`.
+2. Inside `/game/csgo/`, locate `gameinfo.gi`.
+3. Create a new line underneath and add `Game    csgo/addons/metamod` under the line `Game_LowViolence    csgo_lv`.
```

Comment:
You are missing something after `underneath`.

Reply:
good spot. updated